### DISCUSSION TITLE
Set custom label color depending on eg. state?

### DISCUSSION THREAD

**Discussion by sammyke007** [cgiesche/streamdeck-homeassistant#242] [Feature Requests]
Hi

I have an Entity (generic) configured for my heater.
In custom labels I have this config:

```

Temp: {{current_temperature}}°
{% if state != "off" %} Vraag: {{temperature}}° {% else %}{% endif %}
{% if state == "auto" %} Auto {% else %} {{state}} {% endif %}
```

<img width="71" alt="image" src="https://github.com/cgiesche/streamdeck-homeassistant/assets/33524412/7a99393e-15e4-433e-917b-292b79d75cbf">

This works great, but is it possible to also change the custom label text or background color depending on the state?
Eg. 
state == on >> orange label text or orange background
state == off >> white label text or black background

**Comment by cgiesche** [comment]
Hi,

I am working on a complete button-image-rendering-component rewrite, which will allow for those kind of features in future (see https://github.com/cgiesche/streamdeck-homeassistant/discussions/243#discussioncomment-8585354). I will think about how to implement your suggestions but it may take some time. (The hard part is to make it easily usable by "normal" users).
